desktop/index.js

```diff
@@ -0,0 +1 @@
+var app = require('./app')();
```

Comment:
Why does this file exist? Do we plan on having other non-desktop index files that do something else? it seems like we should probably just include `./app` wherever it is we include `desktop`